bigflow/workflow.py

```diff
@@ -417,3 +420,55 @@ def _parse_runtime_str(runtime: str) -> dt.datetime:
         except ValueError:
             pass
     raise ValueError("Unable to parse 'runtime' %r" % runtime)
+
+
+def load_workflows_in_packages(
+        project_dir: Path,
+        packages: List[str]
+) -> Iterator[Any]:
```

Comment:
If you load workflows then it should be `Iterator[Workflow]`, am I right?

And I think that we don't need that docstring, it's basically describing the function signature.

Reply:
Yeah, this is a leftover from previous version that was able to transform workflows to anything.